ringpop.go

```diff
@@ -96,6 +96,25 @@ func mergeDefault(opts *Options) *Options {
 	return opts
 }
 
+type RingpopInterface interface {
+	Destroy()
+	Destroyed() bool
+	App() string
+	WhoAmI() string
+	Uptime() time.Duration
+	RegisterListener(l EventListener)
+	Bootstrap(opts *BootstrapOptions) ([]string, error)
+	HandleEvent(event interface{})
+	Checksum() uint32
```

Comment:
Interesting. I didn't know it was already exposed. Let's avoid breaking API compatibility for for this function and ignore it for now.